src/controller.rs

```diff
@@ -289,9 +290,12 @@ describe! controller {
     before_each {
         use stubs::service::ServiceStub;
         use tls::TlsOption;
+        use profile_service::ProfilePath;
 
         let service = ServiceStub;
-        let controller = FoxBox::new(false, Some("foxbox".to_owned()), 1234, 5678, TlsOption::Disabled);
+        let controller = FoxBox::new(
+          false, Some("foxbox".to_owned()), 1234, 5678,
+                      TlsOption::Disabled, ProfilePath::Default);
```

Comment:
Hmmm, this isn't the convention I've seen. Do you guys have a style guide somewhere?